modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -682,6 +682,16 @@ TEST_P(Test_ONNX_layers, Compare_LT)
     testONNXModels("less");
 }
 
+TEST_P(Test_ONNX_layers, Compare_GTorEQ)
+{
+    testONNXModels("greater_or_equal");
```

Comment:
Looking at `Compare_LT` one might assume that the same skips should be performed for the inference engine. Did you test it?

Reply:
It could be ignored as we don't have regular stable CI with this configuration.

Reply:
Currently, all compare operators have been mapped to `nary_eltwise_layers.cpp`. And `nary_eltwise_layers.cpp` has no inference engine supported.